presto-main/src/main/java/io/prestosql/sql/planner/PlanOptimizers.java

```diff
@@ -226,7 +226,6 @@ public PlanOptimizers(
         Set<Rule<?>> predicatePushDownRules = ImmutableSet.of(
                 new MergeFilters(metadata));
 
-        // TODO: Once we've migrated handling all the plan node types, replace uses of PruneUnreferencedOutputs with an IterativeOptimizer containing these rules.
```

Comment:
But won't it come under the umbrella #811

Reply:
Agreed. This is why i am saying this in the cmt msg:

> While we want to have symbol pruning to be rule-based, the TODO per-se
was misleading.